docs/how-to-use-monty/tutorials/using-monty-for-robotics.md

```diff
@@ -51,13 +50,20 @@ For the first test, we plan to recognize 3D objects inside a phantom (a clear ba
 
```

Comment:
I can't suggest a change here since the line wasn't change but could you change "we plan to recognize" to "we recognize"? and also "we could" to "we can"